components/server/src/authorization/authorizer.ts

```diff
@@ -520,18 +520,26 @@ export class Authorizer {
     }
 }
 
-async function getSubjectFromCtx(passed: Subject): Promise<SubjectId> {
+export async function getSubjectFromCtx(passed: Subject): Promise<SubjectId> {
```

Comment:
nit: this method looks a bit grown. would be good to simplify the logic a bit. maybe also adding a comment that this is only exported for testing could make sense, to avoid future us from calling it

Reply:
I think it's fine, and not that many ways to make it nicer. :slightly_smiling_face: 
Also, it's super-temporary (as in: until end of next week), and well-tested now.

Reply:
Yes, if we are removing it soon, it's absolutely fine 👍